nucliadb_protos/nodereader.proto

```diff
@@ -202,39 +202,88 @@ message VectorSearchResponse {
     int32 result_per_page = 5;
 }
 
-// Relation filters are used to make the 
-// search domain smaller. By providing filters the 
-// search may  be faster.
-message RelationFilter {
-    // Will filter the search to nodes of type ntype.
-    utils.RelationNode.NodeType ntype = 1;
-    // Additionally the search can be even more specific by 
+message RelationNodeFilter {
+    utils.RelationNode.NodeType node_type = 1;
+
+    // Additionally the search can be even more specific by
+    // providing a subtype. The empty string is a wilcard that 
+    // indicates to not filter by subtype. 
+    string subtype = 2;
```

Comment:
Instead of using an empty string, it would be more idiomatic to use an optional here, right ?

Reply:
Totally agree